deploy/determined_deploy/local/cluster_utils.py

```diff
@@ -90,7 +90,7 @@ def docker_compose(
         for extra_file in extra_files:
             base_command += ["-f", extra_file]
     args = base_command + args
-    subprocess.run(args, env=process_env)
+    subprocess.check_call(args, env=process_env)
```

Comment:
doesn't check call just raise a CalledProcessError with the return code? do we need the stdout/stderr from the call for the issue to be clear?

Reply:
both `run` and `check_call` will print stdout/stderr, so this won't change what's printed, you'd still see the underlying docker-compose logs.

so what I've encountered is that docker-compose would fail e.g. when docker daemon is down, or it can't find an image. with `run`, code just continues to roll and fails later on in an odd place. we should fail as soon as docker-compose fails instead, and that's what this `check_call` does.